params/config_mordor.go

```diff
@@ -70,5 +71,9 @@ var (
 		ECIP1017EraRounds:  big.NewInt(2000000),
 		ECIP1010PauseBlock: nil,
 		ECIP1010Length:     nil,
+		RequireBlockHashes: map[uint64]common.Hash{
+			840013: common.HexToHash("0x2ceada2b191879b71a5bcf2241dd9bc50d6d953f1640e62f9c2cee941dc61c9d"),
```

Comment:
I'm not sure these should be in here.

An arguable convenience for a testnet in a tough spot. Maybe let's just consider this temporary, and then eventually remove it once the canonical chain gets a little longer.